src/sentry/monitors/constants.py

```diff
@@ -6,7 +6,7 @@
 MAX_TIMEOUT = 40_320
 
 # Format to use in the issue subtitle for the missed check-in timestamp
-SUBTITLE_DATETIME_FORMAT = "%b %d, %I:%M %p"
+SUBTITLE_DATETIME_FORMAT = "%b %d, %I:%M %p %Z"
```

Comment:
Can we show a matching human readable timezone (`PDT`) instead of `+07:00` offset?

Reply:
Isn't that what `%Z` is?

Reply:
```
In [3]: timezone.now().strftime('%b %d, %I:%M %p %Z')
Out[3]: 'Oct 25, 10:37 PM UTC'
```

Reply:
Ah it is. Got confused by the docs. All good thx